pkg/registry/apis/datasource/register.go

```diff
@@ -69,7 +69,8 @@ func RegisterAPIService(
 	all := pluginStore.Plugins(context.Background(), plugins.TypeDataSource)
 	ids := []string{
 		"grafana-testdata-datasource",
-		//	"prometheus",
+		"prometheus",
+		"graphite",
```

Comment:
Exposing in embedded mode makes testing query service delegation easier (I have been commenting out the condition check)